csrc/mmdeploy/codebase/mmcls/linear_cls.cpp

```diff
@@ -18,6 +18,7 @@ class LinearClsHead : public MMClassification {
  public:
   explicit LinearClsHead(const Value& cfg) : MMClassification(cfg) {
     if (cfg.contains("params")) {
+      softmax_ = cfg["params"].value("softmax", false);
```

Comment:
`with_softmax` looks better to me. Just like `with_argmax` in segmentation.

Reply:
The parameter indicates the class should apply `softmax` on its input, however, `with_argmax` in segmentation indicates `argmax` is applied in the model, which has very different semantics :joy: